proposals/relaxed-simd/Overview.md

```diff
@@ -70,20 +70,27 @@ The specification is updated with the idea of "Relaxed operations":
 
 ## Instructions
 
+`IMPLEMENTATION_DEFINED_ONE_OF(...)` returns any one of the results in the argument list, depending on the implementation.
+
 ### Relaxed swizzle
 
 - `relaxed i8x16.swizzle(a : v128, s : v128) -> v128`
 
-`relaxed i8x16.swizzle(a, s)` selects lanes from `a` using indices in `s`, indices in the range `[0,15]` will select the `i`-th element of `a`, the result for any out of range indices is implementation-defined (i.e. if the index is `[16-255]`.
+`relaxed i8x16.swizzle(a, s)` selects lanes from `a` using indices in `s`,
+indices in the range `[0,15]` will select the `i`-th element of `a`, the result
+for any out of range indices is implementation-defined (i.e. if the index is
+`[16-255]`.
 
 ```python
-def relaxed_i8x16_swizzle(a, s):
+def relaxed_i8x16_swizzle(a : i8x16, s : i8x16):
     result = []
     for i in range(16):
         if s[i] < 16:
             result[i] = a[s[i]]
+        else if s[i] < 128:
+            result[i] = IMPLEMENTATION_DEFINED_ONE_OF(0, a[s[i]%16])
         else:
-            result[i] = UNDEFINED
+            result[i] = 0
```

Comment:
It took me a second to parse this but I think it does make sense: on x86, for `s[i] < 128`, use the lower bits to select a lane; if `s[i] >= 128`, then the upper bit is set and the lane is zeroed.

Reply:
exactly!